Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -37,34 +35,44 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
         self.editDelegate = view
         return view
     }
-    
-    func getCell( row: Int,  delegate: BillingFormTextFieldCellDelegate?) -> UITableViewCell {
-        let currentStyle = style.cells[row]
-        guard style.cells.count > row,
-              var cellStyle = currentStyle.style
+
+    func getCell(row: Int,  delegate: CKOCellTextFieldDelegate?) -> UITableViewCell {
+        guard style.cells.count > row, var viewStyle = style.cells[row].style
```

Comment:
Every cell should be dequeued from the tableview. We shouldn't just create cells from scratch every time.

Reply:
Updated